README.md

```diff
@@ -1,2 +1,75 @@
 # Countries
 This app displays exciting information and facts from countries all over the world
+
+## Built With
+
+- REACT.JS
+- REDUX
+- TAILWIND-CSS
+
+## Install
+### Run the project in your local machine
+
+- Clone the repository
+```
+  $ git clone https://github.com/Estherstarr/countriesAPI.git
+  $ cd countriesAPI
+```
```

Comment:
- [ ] It's required in this project to deploy the application using Heroku, Netlify or github Pages and put the live link in readme file.